src/components/footer.js

```diff
@@ -56,311 +56,118 @@ class Footer extends Component {
 
   render() {
     return (
-      <footer className="dark-footer">
+      <footer className="light-footer">
         <div className="container">
           <div className="row">
             <div className="col-12 col-md-5 col-lg-6">
```

Comment:
Let's widen this column on `md` screens.

Reply:
check newest commit